src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -24,12 +24,49 @@ interface OriginalDocumentProps extends v2.OpenAttestationDocument {
   };
 }
 
+interface ExtensionIconProps {
+  src: string;
+}
+
+const ExtensionIcon: FunctionComponent<ExtensionIconProps> = ({ src }: ExtensionIconProps) => {
+  return (
+    <img
+      src={`/static/images/fileicons/${src}.svg`}
+      className="flex items-center justify-center mr-2"
+      data-testid={`attachment-icon-${src}`}
+    />
+  );
+};
+
+export const getExtension = (mimeType: string | undefined): React.ReactNode => {
+  switch (true) {
+    case mimeType === "text/csv":
+      return <ExtensionIcon src={"csv"} />;
+    case mimeType === "application/vnd.openxmlformats-officedocument.wordprocessingml.document" ||
+      mimeType === "application/msword":
+      return <ExtensionIcon src={"doc"} />;
+    case mimeType === "image/jpeg":
+      return <ExtensionIcon src={"jpg"} />;
+    case mimeType === "image/png":
+      return <ExtensionIcon src={"png"} />;
+    case mimeType === "application/pdf":
+      return <ExtensionIcon src={"pdf"} />;
+    case mimeType === "text/plain":
+      return <ExtensionIcon src={"txt"} />;
```

Comment:
let's make the props abit more explicit by separating src and data-testid props, pass in another prop call `testid` maybe:

< ExtensionIcon src="/static/images/fileicons/doc.svg" testid="attachment-icon-doc" />

Reply:
or even better, props forward them:

```
const ExtensionIcon: FunctionComponent<ExtensionIconProps> = ({ ...props }) => {
  return <img {...props} className="flex items-center justify-center mr-2" />;
};
```